apps/api/src/support/dto/create-campagin-report.dto.ts

```diff
@@ -0,0 +1,48 @@
+import { Prisma, ReportReason, ReportStatus } from '.prisma/client'
+import { Expose, Type } from 'class-transformer'
+import { IsEnum, IsNotEmpty, IsObject, IsString, IsUUID, ValidateNested } from 'class-validator'
+import { ApiProperty } from '@nestjs/swagger'
+import { CreatePersonDto } from '@podkrepi-bg/podkrepi-types'
+
+@Expose()
+export class CreateCampaignReportDto {
+  @ApiProperty()
+  @IsNotEmpty()
+  @Expose()
+  @IsObject()
+  @ValidateNested()
+  @Type(() => CreatePersonDto)
+  public readonly person: CreatePersonDto
+
+  @ApiProperty()
+  @Expose()
+  @IsNotEmpty()
+  @IsUUID()
+  public readonly campaignId: string
+
+  @ApiProperty()
+  @Expose()
+  @IsNotEmpty()
+  @IsString()
+  public readonly reportContent: string
```

Comment:
I feel like if I am reporting something I wouldn't necessarily want to write down anything since we already have a reason, but I don't think it's too big of a deal.

Reply:
Do you think removing the `@IsNotEmpty()` decorator is going to solve it, so the frontend can send an empty string if the person does not enter any content?

Reply:
I am not sure, but in my opinion that would be a better user experience and we should make it behave like that.
Many of the times I report something on websites I just give it a reason and report it. 

For example on YouTube they have like many **well described reasons** to report for and you rarely need to describe further what you meant!